### ISSUE TITLE
Easy to access control to disable/enable Discord RPC

### ISSUE THREAD

**Issue by Davoleo** [clandrew/wdrp#19]
Winamp RPC tends to override any other RPC that is started later than it.
It'd be useful to be able disable and enable the RPC with an easy to access option without having to go into Winamp Preferences

**Comment by clandrew** [comment]
I'm trying to think, what would be easier? A hotkey?

**Comment by Davoleo** [comment]
yeah a hotkey or a button on Winamp gui would work just fine (although I don't know easy that would be to do the second one knowing that Winamp GUI is completely customized)

Another option could be a configuration option in the plugin settings to hide the RPC when Winamp is paused.

**Comment by clandrew** [comment]
Hiding the RPC when Winamp is paused is definitely doable. Actually come to think of it, someone had a PR to do just that but they ran into Git problems and didn't merge it. I think I can merge it for them... will check on that.

More generally, plug-ins changing Winamp UI seems a bit sketchy, hotkey might be doable if it's absolutely needed though
